code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -61,25 +66,44 @@
 		return FALSE
 	if(!ishuman(M))
 		return FALSE
-	if(M.size_multiplier > size_multiplier)
-		to_chat(M,"<span class='warning'>This isn't a pony show! They need to be bigger to ride.</span>")
+	if(M in buckled_mobs)
+		return FALSE
+	if(M.size_multiplier > size_multiplier * 1.2)
+		to_chat(M,"<span class='warning'>This isn't a pony show! You need to be bigger for them to ride.</span>")
 		return FALSE
+	if(M.loc != src.loc)
+		if(M.Adjacent(src))
+			M.forceMove(get_turf(src))
 
 	var/mob/living/carbon/human/H = M
 
 	if(isTaurTail(H.tail_style))
-		to_chat(H,"<span class='warning'>Too many legs. TOO MANY LEGS!!</span>")
+		to_chat(src,"<span class='warning'>Too many legs. TOO MANY LEGS!!</span>")
 		return FALSE
 
 	. = ..()
 	if(.)
 		buckled_mobs[M] = "riding"
 
-/mob/living/carbon/human/MouseDrop_T(mob/living/M, mob/living/user)
-	if(can_buckle && istype(M))
-		if(user_buckle_mob(M, user, silent = TRUE))
-			visible_message("<span class='notice'>[M] starts riding [name]!</span>")
-			return TRUE
+/mob/living/carbon/human/MouseDrop_T(mob/living/M, mob/living/user) //Prevention for forced relocation caused by can_buckle. Base proc has no other use.
+	return
```

Comment:
Um, this will prevent you from ever dragging and dropping anything onto humans, ever, period.

Like dragging ghosts into bodies which is a thing admins do.

Reply:
This might also just prevent stripping entirely, now that I think about it.

Reply:
Luckily Stripping is implemented in  `/mob/MouseDrop()` instead of `MouseDrop_T()`

Reply:
Yeah, everything is called from MouseDrop(). Only buckling uses MouseDrop_T()

Reply:
@Verkister I'm pretty sure it's fine but can you test this in a test server with these changes?

Reply:
No issues found with our current features. May have a possible need for little tweaks in the future though, but only if some future feature ends up requiring human targeted MouseDrop_T()